src/app/locationForm/common/FormView.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ConfirmationOptions from '../ConfirmationOptions';
```

Comment:
If we have a /common directory now, I'd think ConfirmationOptions should be in it too.

Reply:
Should be fixed in 2398171c5501714a4ace402c79c4aa518cae2e2e